xwiki-platform-tools/xwiki-platform-tool-jetty/xwiki-platform-tool-jetty-resources/src/main/resources/stop_xwiki.bat

```diff
@@ -50,4 +50,4 @@ set JETTY_OPTS=%JETTY_OPTS% STOP.KEY=xwiki STOP.PORT=%JETTY_STOP_PORT%
 
 REM Pause so that the command window used to run this script doesn't close automatically in case of problem
 REM (like when the JDK/JRE is not installed)
-PAUSE
```

Comment:
Thanks for the PR.

Not sure about the removal of the new line at the end. Could you put that as it was (unless you have a good reason to remove it) and I'll apply the PR? thx